### DISCUSSION TITLE
Hosting my website online

### DISCUSSION THREAD

**Discussion by ghost** [community/community#49081] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

hi.. I have followed all proper steps and uploaded my files to the new repository. But the thing is when i go to settings and click Pages, i don't see the part that gives me a url to me website.. how come

**Comment by Ntshangase** [comment]
Hello, Where were you trying to host from?,
 
I suggest you double check that you followed everything correctly, sometime you can miss something small.

 You can also check out GitHub Pages https://docs.github.com/en/pages/quickstart
